lib/matplotlib/axes/_axes.py

```diff
@@ -6464,7 +6442,13 @@ def pcolormesh(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
         collection._scale_norm(norm, vmin, vmax)
 
         coords = coords.reshape(-1, 2)  # flatten the grid structure; keep x, y
+        self._update_pocolor_lims(collection, coords)
+        return collection
 
+    def _update_pocolor_lims(self, collection, coords):
```

Comment:
Minor typo?
```suggestion
    def _update_pcolor_lims(self, collection, coords):
```

Reply:
Woops, thanks for catching! Should be fixed now